### DISCUSSION TITLE
Every minute request?

### DISCUSSION THREAD

**Discussion by maboxx** [qdm12/ddns-updater#308] [Q&A]
I have the following configuration which I have understood so that there is only every 5 minutes a query about my DynDNS address.
`    environment:
      - CONFIG=
      - PERIOD=5m
      - UPDATE_COOLDOWN_PERIOD=5m
      - PUBLICIP_FETCHERS=all
      - PUBLICIP_HTTP_PROVIDERS=opendns,ifconfig
      - PUBLICIPV4_HTTP_PROVIDERS=all
      - PUBLICIPV6_HTTP_PROVIDERS=all
      - PUBLICIP_DNS_PROVIDERS=cloudflare
      - PUBLICIP_DNS_TIMEOUT=3s
      - HTTP_TIMEOUT=10s`
      - 
But in my Pi-hole I can see that there is every minute a call to my DynDNS address. Is that correct?

EDIT: I'm sorry, I only now see that the question already exists, [here](https://github.com/qdm12/ddns-updater/discussions/303). Please close this Question!

**Comment by qdm12** [comment]
Answered in #303 it's due to the healthcheck!
